app/views/comments/_form.html.erb

```diff
@@ -23,7 +23,8 @@
 
   <div class="field">
     <%= form.label :dependency_id %>
-    <%= form.text_field :dependency_id %>
+    <%= select_tag(:dependency_id,
```

Comment:
Please use `form.select` instead of `select_tag`, specially in rails 5.2 with the new `form_with` method.